Update index.md (#4459)
SumoLogic/sumologic-documentation: docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/python/index.md

opentelemetry-instrument python3 SCRIPT_NAME.py
```

> For Auto-instrumentation to work, you will need to include `opentelemetry-instrument` at the beginning of your application execution. If this is not an option for you, then you will need to follow the usage steps for each instrumentation library you install. For example, for Django instrumentation, you will need to add the following two lines to your code (outlined in the [Usage section](https://opentelemetry-python-contrib.readthedocs.io/en/latest/instrumentation/django/django.html#id1) of the library documentation):

Reviewer: ```suggestion
To enable auto-instrumentation, you'll need to include `opentelemetry-instrument` at the beginning of your application execution. If this is not an option for you, you’ll need to follow the usage instructions for each instrumentation library you install.  For example, for Django instrumentation, you will need to add the following two lines to your code (outlined in the [Usage section](https://opentelemetry-python-contrib.readthedocs.io/en/latest/instrumentation/django/django.html#id1) of the library documentation):
```

```
from opentelemetry.instrumentation.django import DjangoInstrumentor

DjangoInstrumentor().instrument()     ##This goes inside of main
```

> Adding these lines prevents you from having to use the `opentelemetry-instrument` command in the execution of your app.

Reviewer: ```suggestion
Adding these lines eliminates the need to use the `opentelemetry-instrument` command in the execution of your app.
```

## Log correlation

* To enable trace context injection into logs, set the `OTEL_PYTHON_LOG_CORRELATION` environment variable to `true` ([learn more](https://opentelemetry-python-contrib.readthedocs.io/en/latest/instrumentation/logging/logging.html#enable-trace-context-injection))

## Examples

### Django Instrumentation

> [This article](https://opentelemetry-python.readthedocs.io/en/stable/examples/django/README.html#django-instrumentation) will show you how to use the `opentelemetry-instrumentation-django` library to automatically instrument a Django app. It will show you how to reference the library manually within your code, as well as how to use [Auto-instrumentation](https://opentelemetry-python.readthedocs.io/en/stable/examples/django/README.html#auto-instrumentation) without having to edit your code.

Reviewer: ```suggestion
[This article](https://opentelemetry-python.readthedocs.io/en/stable/examples/django/README.html#django-instrumentation) will show you how to use the `opentelemetry-instrumentation-django` library to automatically instrument a Django app. It covers how to reference the library manually within your code, as well as how to use [Auto-instrumentation](https://opentelemetry-python.readthedocs.io/en/stable/examples/django/README.html#auto-instrumentation) without having to edit your code.
```

### Sample Flask App Instrumentation

> [This article](https://opentelemetry.io/docs/languages/python/getting-started/) will walk you through deploying your own simple Flask app and configuring instrumentation on that app. The example will log the traces to the console for simple testing.

Reviewer: ```suggestion
[This article](https://opentelemetry.io/docs/languages/python/getting-started/) walks you through deploying a simple Flask app and configuring instrumentation for it. The example demonstrates how to log traces to the console for basic testing.
```
